README.md

```diff
@@ -34,3 +34,16 @@ Now you can run tests, using forge:
 ```bash
 forge test
 ```
+
+## Specifications
```

Comment:
```suggestion
## Limitations
```

I think we should name this in some way that it captures the reader's attention, rather than let them skip this section. This section must be read by the maximum number of readers